### ISSUE TITLE
Overlapping APIs between train and dev set

### ISSUE THREAD

**Issue by ypruan** [google-research-datasets/dstc8-schema-guided-dialogue#1]
Hi, I find there're overlapping APIs between train/dev set, like Hotels_1, Music_1, Events_1, so does it conflict with the zero-shot learning (scalable to unseen APIs)?

**Comment by abhirast** [comment]
Hi. Thank you for your comment. We have 17 services/APIs in the dev set, out of which 7 are also present in the training set. We decided to do this so that we can evaluate the systems in both these settings. The test set will similarly have unseen APIs as well as some that are shared with the train/dev set.
